polyfills/Array/from/config.json

```diff
@@ -7,11 +7,12 @@
 	"browsers": {
 		"chrome": "<45",
 		"firefox": "4 - 31",
-		"ie": "6 - *",
+		"ie": "6 - 11",
```

Comment:
We are currently using an older version of the [useragent](https://github.com/3rd-Eden/useragent) module which identifies Edge as IE12. We also alias Edge to IE in our [ua.js](https://github.com/Financial-Times/polyfill-service/blob/db69e86e91dc7e12df4e09e20919a9799c93ae92/lib/UA.js#L70) module.

I don't believe IE development to be carrying on, I think it may be fine to leave this as `6 - 11`.